distribution/docs/src/main/resources/_contents/_ui-contents/managing-ui-contents.adoc

```diff
@@ -179,8 +179,13 @@ Admin Console.
  -TMS (TileMapService), +
  -GE (GoogleEarth), +
  Examples: +
- -{"type" "WMS" "url" "http://example.com" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5} +
- -{"type" "WMT" "url" "http://example.com" "layer" "layer1" "style" "default" "tileMatrixSetID" "tileSetId"}
+ -{"type" "OSM" "url" "http://example.osm.org" "fileExtension" "png" "alpha" 0.3 } +
+ -{"type" "AGM" "url" "http://example.agm.org"} +
+ -{"type" "BM" "url" "https://example.bm.org" "key" "API KEY"} +
+ -{"type" "WMS" "url" "http://example.wms.org" "layers" ["layer1" "layer2"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5} +
```

Comment:
It should depend on the type of Imagery Provider you are using.  Multiple layers work in a single imageryProvider for WMS (which the example shows), but an ArcGis provider only takes a URL.